Makefile

```diff
@@ -34,6 +36,12 @@ atomfs-cover: BUILDCOVERFLAGS=-cover
 gotest: $(GO_SRC)
 	go test -coverprofile=unit-coverage.txt -ldflags "$(VERSION_LDFLAGS)"  ./...
 
+$(PRE_EROFS_STACKER):
+	mkdir -p $(TOOLS_D)/bin
+	wget --progress=dot:giga https://github.com/project-stacker/stacker/releases/download/$(PRE_EROFS_STACKER_VERSION)/stacker
```

Comment:
This can use `--output-document $(TOOLS_D)/bin/pre-erofs-stacker`

Reply:
done